lib/parse/data/parseAudioData.spec.ts

```diff
@@ -5,15 +5,15 @@ describe('lib/parse/data', () => {
   describe('parseAudioData', () => {
     const mockRssItem: IRssItem = {
       guid: 'foo-bar',
-      link: '//foo.com/foo-bar',
+      link: 'http://foo.com/foo-bar',
```

Comment:
Your test fixtures were just all missing a scheme for some reason?

Reply:
yeah. I had initial written them without it because my fingers are lazy by default.